### PR TITLE
migrating the website fromgschool.ydgkim.com/saspowerschool to github

### PR DESCRIPTION
**Pull request by jadijadi**
Dear Gary, I've opened a new PR to keeps things clean. This PR includes changes you proposed on the previous PR and resolves https://github.com/gary-kim/saspes/issues/17 .

Do not hesitate to comment any changes you see required.

Signed-off-by: Jadi <jadijadi@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,8 +52,7 @@ People have gotten the extension to work on the following browsers
 
 ## Download
 
-Downloads for both Firefox and Chromium can be found on the project website: [https://gschool.ydgkim.com/saspowerschool/](https://gschool.ydgkim.com/saspowerschool/?pk_campaign=SASPES&pk_kwd=github-readme)
-
+Download for Chromium can be found on [chrome web store](https://chrome.google.com/webstore/detail/sas-powerschool-enhanceme/ehnkngeidilnoabcjjimkomcggndbhnk) and for Firefox, please check the [relase directory](https://github.com/gary-kim/saspes/releases/).
```
Comment (gary-kim) [inline README.md]:
```suggestion
Download for Chromium can be found on [chrome web store](https://chrome.google.com/webstore/detail/sas-powerschool-enhanceme/ehnkngeidilnoabcjjimkomcggndbhnk) and for Firefox, please check [releases](https://github.com/gary-kim/saspes/releases/).
```

**Comment by jadijadi** [comment]
Dear kim, I've did the changes, rebased, singed-off and pushed with force. Hope it is fine now. Do not hesitate to propose any fixes needed.

**Comment by gary-kim** [comment]
Thank you for your PR!

**Comment by jadijadi** [comment]
Thank you too. I really liked the idea of a plugin, adding more insight into a page :)

**Pull request merged**